keras/backend/tensorflow_backend.py

```diff
@@ -4220,13 +4241,16 @@ def range_less_than(_, current_input):
     label_ind = tf.boolean_mask(label_array, dense_mask)
 
     batch_array = tf.transpose(tf.reshape(tf.tile(tf.range(label_shape[0]),
-                                                  max_num_labels_tns), reverse(label_shape, 0)))
+                                                  max_num_labels_tns),
+                                          reverse(label_shape, 0)))
     batch_ind = tf.boolean_mask(batch_array, dense_mask)
-    indices = tf.transpose(tf.reshape(concatenate([batch_ind, label_ind], axis=0), [2, -1]))
+    indices = tf.transpose(tf.reshape(concatenate([batch_ind, label_ind], axis=0),
+                                      [2, -1]))
 
     vals_sparse = tf.gather_nd(labels, indices)
 
-    return tf.SparseTensor(tf.to_int64(indices), vals_sparse, tf.to_int64(label_shape))
+    return tf.SparseTensor(tf.to_int64(indices),
```

Comment:
please shorten the line by using temporary variables, it will be easier to read.
```python
indices = tf.to_int64(indices)
...
```